modules/electromagnetics/test/tests/benchmarks/eigenvalue_problems/eigen_base.i

```diff
@@ -0,0 +1,138 @@
+# Base input file for eigenvalue example tests for multiple waveguide geometries
+# RECTANGULAR (Default)
+#     Mesh file rectangular.e based on Mesh block:
+#         [Mesh]
+#           type = GeneratedMesh
+#           dim = 2
+#           nx = 50
+#           ny = 25
+#           xmin = 0
+#           xmax = 2
+#           ymin = 0
+#           ymax = 1
+#           elem_type = TRI3
+#         []
+#     Expected analytic eigenvalue = 12.337005
+#     EM Module calculated eigenvalue = 12.363806
+# CIRCULAR (Mesh/file=circle.msh, BCs/active='circle eigen_circle')
+#     Mesh generated using gmsh
+#       radius = 1
+#       center = (0, 0)
+#     Expected analytic eigenvalue = 5.784025
+#     EM Module calculated eigenvalue = 5.824152
```

Comment:
It's been a while since I last ran this case, but I believe so. For the purpose of the test I didn't want the mesh to be too fine (so that it was faster).